config/config.js

```diff
@@ -56,26 +51,20 @@ var getGlobbedPaths = function(globPatterns, excludes) {
  * Validate NODE_ENV existance
  */
 var validateEnvironmentVariable = function() {
-    glob('./config/env/' + process.env.NODE_ENV + '.js', {
-        sync: true
-    }, function(err, environmentFiles) {
-        console.log();
-
-        if (!environmentFiles.length) {
-            if (process.env.NODE_ENV) {
-                console.error(chalk.red('No configuration file found for "' + process.env.NODE_ENV + '" environment using development instead'));
-            } else {
-                console.error(chalk.red('NODE_ENV is not defined! Using default development environment'));
-            }
-
-            process.env.NODE_ENV = 'development';
+    var environmentFiles = glob.sync('./config/env/' + process.env.NODE_ENV + '.js');
+    console.log();
+    if (!environmentFiles.length) {
+        if (process.env.NODE_ENV) {
+            console.error(chalk.red('No configuration file found for "' + process.env.NODE_ENV + '" environment using development instead'));
         } else {
-            console.log(chalk.bold('Application loaded using the "' + process.env.NODE_ENV + '" environment configuration'));
+            console.error(chalk.red('NODE_ENV is not defined! Using default development environment'));
```

Comment:
The github diff is a little messed up. It makes it look like I changed that line from console.log to console.error, when really all that changed is indentation. 

If you would like me to change it to either `warn` or `log` though I can.